core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/CollectInformation.scala

```diff
@@ -276,11 +283,27 @@ object CollectInformation extends Logging {
             val filtered = accums.filter { a =>
               stageIdsForSQL.contains(a.stageId)
             }
-            val accumValues = filtered.map(_.value.getOrElse(0L))
-            if (accumValues.isEmpty) {
-              None
+            // If metricType is size, average or timing, we want to use the update value to get the
+            // min, median, max, and total. Otherwise, we want to use the value.
```

Comment:
Nit: The comment can be more readable putting quotes around the code parts
```suggestion
            // If metricType is size, average or timing, we want to read field `update` value to get the
            // min, median, max, and total. Otherwise, we want to use field `value`.
```